graphics/feh/Portfile

```diff
@@ -50,6 +51,11 @@ build.args          PREFIX=${prefix} \
                     'exif=1' \
                     'verscmp=0'
 
+# filelist.c:255:40: error: passing argument 3 of 'scandir'
+# from incompatible pointer type [-Wincompatible-pointer-types]
+configure.cflags-append \
+                    -Wno-incompatible-pointer-types
```

Comment:
we really don't want to add this to every port to fix building with GCC14. That is something that should be fixed globally...

Reply:
I mean, I would prefer having it added centrally, but no idea if it can be done.

We had similar problem with clang-18, and I think all those `-Wno-error=` flags were added per-port.